plugins/src/lib.rs

```diff
@@ -215,6 +215,19 @@ impl Parse for DeriveSerde {
     }
 }
 
+/// Generates:
+/// - derive of Debug, serde Serialize & Deserialize
+/// - serde crate annotation
+#[proc_macro_attribute]
```

Comment:
I think this should be a `proc_macro_derive` -- https://doc.rust-lang.org/reference/procedural-macros.html#derive-macros

Reply:
From what I've read derive macros allow you to append something to a strict/enum, so that wouldn't allow me to just substitute the 2 needed attributes.

I might be able to do it with a derive, but I'd think id be copying serdes implementation of that macro then.

I haven't seen how to do macro aliasing with derive macros